### DISCUSSION TITLE
Add judgment type and better in REPL print debugging

### DISCUSSION THREAD

**Discussion by d1y** [vlang/v#11605] [Ideas]
```console
>>> import http
>>> http.get('https://h5.m.taobao.com/ocean/privatenode/shop.html?&sellerId=50852803')
>>> fc := http.get('https://h5.m.taobao.com/ocean/privatenode/shop.html?&sellerId=50852803')
>>> println(fc)
.vrepl.v:4 `?http.Response` needs to have method `str() string` to be printable
```

Currently in the REPL can only be converted to str and then printed, which is very difficult. And, I want to be able to add a value function that can see all the properties, just like in python `dir()`

```console

>>> import os
>>> dir(os)

```

I hope to solve the problem of putting the cache file under /tmp when running the file. Otherwise, the cache file will be added to git when compiling. This is too uncomfortable.

**Comment by spytheman** [comment]
About the tmp.c files: PR #1743 will make git ignore all *.tmp.c files inside this repo.

**Comment by d1y** [comment]
> About the tmp.c files: PR #1743 will make git ignore all *.tmp.c files inside this repo.

run `.v` file and REPL runtime Will create a bunch of meaningless files.
Is creating a cache file in the current directory, which is very difficult

**Comment by spytheman** [comment]
You mean the .vrepl.v file ?

**Comment by d1y** [comment]
> You mean the .vrepl.v file ?

Yes
